### DISCUSSION TITLE
Finer Mesh Paraview Problem

### DISCUSSION THREAD

**Discussion by oguzkaann** [mdolab/dafoam#288] [General]
Hello sir,

I applied finer mesh like you posted at Dafoam FAQ. ı could increased mesh cell number. But ı could not visualize on Paraview. There was error and ı did not understand. Could you help me sir?

![image](https://user-images.githubusercontent.com/95033071/156925077-684c1b40-a660-4ddd-9e80-2ff3a6dc4d07.png)

**Comment by friedenhe** [comment]
Make sure you don't have special chars or a space in your folder/file names.
